ogb_lsc/mag/organize_data.sh

```diff
@@ -46,22 +46,23 @@ if [[ -z "${TASK_ROOT}" ]]; then
   exit 1
 fi
 
-DATA_ROOT="${TASK_ROOT}"/data
-
+DATA_ROOT="${TASK_ROOT}/data/mag240m_kddcup2021"
```

Comment:
I think if we want to match the structure of the data that is currently store in the bucket, then this additional "/mag240m_kddcup2021"suffix is not necessary?

E.g. directories are something like ".../mag/data/raw/..."

Reply:
So, we should also modify one relevant part, which is located in README.md. We could change the "Once this completes, a new directory ROOT/mag240m_kddcup2021/raw will be created" to be "Once this completes, a new directory ROOT/data/raw will be created", right?

Reply:
Yes, you are right, and perhaps also:

we can change:
`/bin/bash organize_data.sh -i ROOT -o ROOT`
to
`/bin/bash organize_data.sh -i DOWNLOAD_DIR -o ROOT`

As I expect that should still work, right?